src/js/components/Chat.jsx

```diff
@@ -96,6 +113,8 @@ class ConnectedChat extends Component {
                                 {/*TODO: Detect if message has image to display in chat.*/}
                                 {/*<CardImg top width="100%" src="https://placeholdit.imgix.net/~text?txtsize=33&txt=318%C3%97180&w=318&h=180" alt="Card image cap" />*/}
                                 <CardBody>
+                                    {recentImage ? <CardImg top height="100%" width="100%" src={imgSource}
```

Comment:
This would make all messages have images

Reply:
Only if the message has a recentImage. Hence el if